pegasus/sites.v3/code.org/views/homepage_below_hero_special2020_banner.haml

```diff
@@ -3,8 +3,7 @@
     .special_2020_announcement{style: "text-align: center; background-color: #0094ca; color: white; font-size: 18px; padding: 16px; overflow: hidden"}
       .container_responsive
         .col-66
-          .icon.tablet-feature{style: "float: left; font-size: 60px; padding-right: 30px"}
-            %i{:class=>"fa fa-home"}
+          %img{src: "/images/homepage/virtual_sync.png", style: "float: left; height: 60px; padding-right: 30px"}
```

Comment:
Should we still include `.tablet-feature` so that this doesn't show on smaller devices?

Reply:
ooh good catch! thank you!

Reply:
Sure thing.  It's definitely worth double-checking that the page looks good at a variety of widths too, especially for the homepage.